index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html lang="fr">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <!-- LOGO -->
+            <div class="logo">My Logo</div>
+            <ul class="nav-links">
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class="body-bar">
+        <div class="align-center-text">
+            <p class="header-text">I'm Abol<br>Glad to see <br>you!</p>
+            <p class="content-text">I'm a software developer
+                <br>build a product feature or website look
+                <br>through some of my work and
+                <br>experience! if you like what you see and
+                <br>have a project you need coded, dont
+                <br>hesistate to contact me
+            </p>
+            <p class="social-connect">LET'S CONNECT</p>
+            <ul class="social-align">
+                <li class="social-media-icons t"><img src="./images/twitter.png" alt=""></li>
+                <li class="social-media-icons"><img src="./images/linkedin.png" alt=""></li>
+                <li class="social-media-icons"><img src="./images/median.png" alt=""></li>
+                <li class="social-media-icons"><img src="./images/git.png" alt=""></li>
+                <li class="social-media-icons"><img src="./images/chill.png" alt=""></li>
+            </ul>
```

Comment:
Let's wrap this section inside the `section` tag. Since it's the project requirement to use [HTML5 semantic tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). That will make the code easier in for `accessibility` purposes.

Reply:
Done